components/Notifications.tsx

```diff
@@ -73,11 +73,10 @@ export default function Notifications() {
                             Copy tx hash
                           </PillAction>
                           <a
-                            className="ml-2"
-                            href={`https://etherscan.io/tx/${hash}`}
+                            href={getBlockExplorerUrl(hash as string, chainId as number)}
                             target="_blank"
                           >
-                            <PillAction icon="external-link">View on Etherscan</PillAction>
+                            <PillAction icon="external-link">View on a block explorer</PillAction>
```

Comment:
nit: notification area is small so we may need the space here. feel free to ignore this one.

```suggestion
                            <PillAction icon="external-link">View on block explorer</PillAction>
```